testing/internal/e2e/tests/static/session_end_delete_user_test.go

```diff
@@ -18,6 +18,8 @@ import (
 // TestCliSessionEndWhenUserIsDeleted tests that an active session is canceled when the respective
 // user who started the session is deleted.
 func TestCliSessionEndWhenUserIsDeleted(t *testing.T) {
+	deleted := false
```

Comment:
Although in this context the meaning of this variable is quite clear, we may want to be more explicit anyway, e.g.:
```suggestion
	isUserDeleted := false
```
or

```suggestion
	userIsDeleted := false
```